deploy/linux/io.github.myst6re.makoureactor.desktop

```diff
@@ -5,7 +5,7 @@ GenericName=FF7 Field Script Editor
 GenericName[fr]=Éditeur d'écrans FF7
 Comment=Can edit field from Final Fantasy VII (PS & PC)
 Comment[fr]=Permet de modifier les écrans de Final Fantasy VII (PS et PC)
-Icon=Makou_Reactor
+Icon=io.github.myst6re.makoureactor
```

Comment:
What happen to the original Makou_Reactor.desktop? not needed anymore?

Reply:
its been renamed to the Reverse FDN scheme. This is required for the appstream metainfo as well as being a unique identifier.